iina/VideoView.swift

```diff
@@ -285,8 +285,12 @@ class VideoView: NSView {
 
     if videoLayer.colorspace != nil {
       Logger.log("Nilling out colorspace", subsystem: hdrSubsystem)
-      videoLayer.colorspace = nil;
       videoLayer.wantsExtendedDynamicRangeContent = false
+      // It seems that once there is a layer with colorspace set to something extended linear,
+      // then setting colorspace to nil will cause falling back to ExtendedLinearsRGB
+      videoLayer.colorspace = CGColorSpaceCreateDeviceRGB()
+      player.mpv.setString(MPVOption.GPURendererOptions.toneMapping, "bt.2390")
```

Comment:
Again, according to mpv document, [the default value of `--tone-mapping` is `auto`](https://[mpv.io](https://mpv.io/manual/master/#options-tone-mapping)/manual/master/#options-tone-mapping). Unless we set these options to the values you specified by default, enabling and disabling HDR may make SDR videos look different.

Reply:
The bundled `libmpv.dylib` version is `0.34.1`, where default value of `--tone-mapping` is `bt.2390`. 

`--tone-mapping=auto` was added in  mpv-player/mpv@a9cb2e2821adb40e7548e8233390e79706104041 and only presented on newer `0.35.0` version.

Reply:
We will eventually upgrade mpv. Does setting the value to an empty string do the trick?

Reply:
Unfortunately it does not work.

```
[  37.562][v][cplayer] Set property: tone-mapping="" -> -4
```

and the option value was not changed